### ISSUE TITLE
[bug] --help doesn't print entire output 

### ISSUE THREAD

**Issue by vsoch** [GoogleContainerTools/container-diff#306]
With the latest release, `container-diff --help` doesn't print anything as it should (see https://github.com/GoogleContainerTools/container-diff/issues/301#issuecomment-478742411)

### Expected behavior

```bash
container-diff --help
container-diff is a CLI tool for analyzing and comparing container images.

Images can be specified from either a local Docker daemon, or from a remote registry.
To specify a local image, prefix the image ID with 'daemon://', e.g. 'daemon://gcr.io/foo/bar'.
To specify a remote image, prefix the image ID with 'remote://', e.g. 'remote://gcr.io/foo/bar'.
If no prefix is specified, the local daemon will be checked first.

Tarballs can also be specified by simply providing the path to the .tar, .tar.gz, or .tgz file.

Usage:
  container-diff [command]

Available Commands:
  analyze     Analyzes an image: [image]
  diff        Compare two images: [image1] [image2]
  help        Help about any command
  version     Print the version of container-diff

Flags:
      --format string      Format to output diff in.
  -h, --help               help for container-diff
  -v, --verbosity string   This flag controls the verbosity of container-diff. (default "warning")

Use "container-diff [command] --help" for more information about a command.
```

### Actual behavior
```bash
$ container-diff --help
Usage of container-diff:
```
### Information
- OS: Tested on ubuntu 18.04, 16.04, and archlinx (see issue linked above)
- container-diff version: v0.15.0

### Steps to reproduce the behavior
 Install container-diff as specific for linux in the README.md, and then run `container-diff --help`.

**Comment by Dhron** [comment]
@vsoch hey, could I try fixing this one? =)

**Comment by vsoch** [comment]
of course! @donmccasland is master and commander, but you also have my dinosaur blessing :sparkles:

**Comment by vsoch** [comment]
Note that the cobra issue linked above is being marked as stale, so likely help won't come from there, unless someone wants to comment / unmark as stale here that is interested in working on this.

**Comment by lplazas** [comment]
@vsoch If still unassigned I'd love to give this one a try.

**Comment by vsoch** [comment]
@lplazas I'd say go for it - it's been a long time since October 19th, and if you have bandwidth, it's a +1 from me!

**Comment by nkubala** [comment]
@lplazas yeah go for it! and @vsoch good to see you still hanging out around here :)

**Comment by vsoch** [comment]
Always! I love containers, although I haven't done work with container-diff recently. People aren't as interested in metadata / provenance -type stuffs as I wish they would be :)

**Comment by nkubala** [comment]
yeah they like to leave the dirty work to us :)

not sure if you're working on Kubernetes stuff, but if you are be sure to check out [skaffold](https://github.com/GoogleContainerTools/skaffold)!

**Comment by lplazas** [comment]
Alright, I'll work on it.

**Comment by lplazas** [comment]
@nkubala @vsoch please take a look at the PR when you have some time.

**Comment by vsoch** [comment]
Will do, probably tomorrow.

**Comment by vsoch** [comment]
All set! Thanks for this fix! I'm glad it turned out to be relatively simple. I'll follow up on the issue I posted on cobra to point to your PR.

**Comment by pelletier2017** [comment]
I am still seeing this same error on Mac Mohave 10.14.6. I'm not sure which version of container-diff I'm using because the flags -v or --version don't seem to exist. I installed it with the command in the readme.

`curl -LO https://storage.googleapis.com/container-diff/latest/container-diff-darwin-amd64 && chmod +x container-diff-darwin-amd64 && sudo mv container-diff-darwin-amd64 /usr/local/bin/container-diff`

**Comment by nkubala** [comment]
@pelletier2017 sorry this is years overdue, but I just released v0.16.0 which contains this fix: https://github.com/GoogleContainerTools/container-diff/releases/tag/v0.16.0

give it a try and let me know if you see any issues!
